keps/prod-readiness/sig-scheduling/3633.yaml

```diff
@@ -3,3 +3,5 @@ alpha:
   approver: "@wojtek-t"
 beta:
   approver: "@wojtek-t"
+stable:
+  approver: "@wojtek-t"
```

Comment:
/assign @wojtek-t 

Please assign it to another person if needed. 🙏

Reply:
this LGTM from PRR perspective - I already approved